librz/bin/mangling/swift-sd.c

```diff
@@ -263,8 +259,8 @@ RZ_API char *rz_bin_demangle_swift(const char *s, bool syscmd) {
 			}
 			break;
 		case 'I': // interfaces
-			/* TODO */
-			break;
+			// TODO
+			return NULL;
```

Comment:
Is this the exact change that fixes test?

Reply:
Yes